rust/src/nfs/nfs4.rs

```diff
@@ -52,6 +52,11 @@ impl NFSState {
             fill_bytes = 4 - pad;
         }
 
+        // linux defines a max of 1mb. Allow several multiples.
+        if w.write_len == 0 || w.write_len > 16777216 {
```

Comment:
If the limit is `1MB` why compare against `16MB`?

Reply:
eheh copied pasted from nfs.rs
@victorjulien do you recall if it is 1 or 16 ?

Reply:
Linux is 1MiB but I didn't find much info about other implementations, so I thought I'd take some multiple.